components/Chat/Chat.js

```diff
@@ -0,0 +1,118 @@
+import PropTypes from 'prop-types';
+
+import {
+  Chat as StreamChat,
+  ChannelHeader,
+  Channel,
+  MessageInputSmall,
+  MessageSimple,
+  MessageList,
+  MessageLivestream,
+  Thread,
+  Window,
+} from 'stream-chat-react';
+
+import { Streami18n } from 'chat';
+import { ConnectionStatus, useChat } from 'chat/ChatProvider';
+
+import { useAuth } from 'providers/AuthProvider';
+import { useModalDispatch, showModal } from 'providers/ModalProvider';
+
+import { Box, Button, Loader } from 'ui-kit';
+
+import Styled from './Chat.styles';
+
+// Shortcuts
+const { CONNECTING, DISCONNECTED, ERROR } = ConnectionStatus;
+
+export default function Chat(props = {}) {
+  const [chatClient, connectionStatus] = useChat();
+  const [{ authenticated }] = useAuth();
+  const modalDispatch = useModalDispatch();
+
+  const connecting = connectionStatus === CONNECTING;
+  const disconnected = connectionStatus === DISCONNECTED;
+  const loading = connecting || disconnected;
+  const error = !props.streamChatChannel || connectionStatus === ERROR;
+
+  if (loading) {
+    return (
+      <Styled.CenteredContent>
+        <Loader
+          alignSelf="center"
+          alignContent="center"
+          text="Connecting chat"
+        />
+      </Styled.CenteredContent>
+    );
+  }
+
+  if (error) {
+    return (
+      <Styled.CenteredContent>
+        <Box as="p" color="alert" textAlign="center" px="base">
+          Sorry, something went wrong!
+          <br />
+          <br />
```

Comment:
We could just do two separate `<p>` tags for this, no?

**_Nitpick Ninja_** ™️